js_modules/dagster-ui/packages/ui-components/src/components/ToggleButton.tsx

```diff
@@ -0,0 +1,22 @@
+import {Icon} from './Icon';
+import {UnstyledButton} from './UnstyledButton';
+
+type Props = {
+  isOpen: boolean;
+  onToggle: (e: React.MouseEvent<HTMLButtonElement>) => void;
+};
+export const ToggleButton = ({isOpen, onToggle}: Props) => {
+  return (
+    <UnstyledButton
+      onClick={(e) => {
+        onToggle(e);
+      }}
+      style={{cursor: 'pointer', width: 18}}
+    >
+      <Icon
+        name="arrow_drop_down"
+        style={{transform: isOpen ? 'rotate(0deg)' : 'rotate(-90deg)'}}
+      />
```

Comment:
Very low priority nit but I'd love to call this "DisclosureTriangleButton" or something like that since it has a hardcoded arrow in it 👀